query/src/main/scala/filodb/query/exec/PromQlRemoteExec.scala

```diff
@@ -44,12 +43,12 @@ case class PromQlRemoteExec(queryEndpoint: String,
   override val urlParams = Map("query" -> promQlQueryParams.promQl)
   private val dummyQueryStats = QueryStats()
 
-  override def sendHttpRequest(execPlan2Span: Span, httpTimeoutMs: Long)
-                              (implicit sched: Scheduler): Future[QueryResponse] = {
+  override def sendRequest(execPlan2Span: Span, httpTimeoutMs: Long)
```

Comment:
there are a couple RemoteExecChild classes? Should we apply the same change to all of them